demos/reorderer/gridReorderer/js/gridReordererDemo.js

```diff
@@ -10,23 +10,20 @@ You may obtain a copy of the ECL 2.0 License and BSD License at
 https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 */
 
-// Declare dependencies
 /* global fluid */
 
-var demo = demo || {};
-
 (function (){
     "use strict";
+    
+    fluid.defaults("demo.initGridReorderer", {
```

Comment:
Is <code>initGridReorderer</code> still the appropriate name for this now that it has been defined as a component grade?